src/semidiscretization.jl

```diff
@@ -46,18 +46,25 @@ end
                        boundary_conditions=boundary_condition_periodic,
                        RealT=real(solver),
                        uEltype=RealT,
-                       initial_cache=(tmp1 = Array{RealT}(undef, nnodes(mesh)),))
+                       initial_cache = (tmp1 = Array{RealT}(undef, nnodes(mesh)),
+                                        tmp_partitioned = ArrayPartition(ntuple(_ -> zeros(real(solver),
+                                                                                           nnodes(mesh)),
+                                                                                 Val(nvariables(equations))))))
 
 Construct a semidiscretization of a PDE.
 """
+
```

Comment:
```suggestion
```
There should be now newline. Otherwise the docstring is not attached to the function and Documenter.jl will not render it correctly.